x/ecocredit/server/core/mint_batch_credits.go

```diff
@@ -118,8 +119,9 @@ func (k Keeper) MintBatchCredits(ctx context.Context, req *core.MsgMintBatchCred
 		}
 	}
 
-	if err := sdk.UnwrapSDKContext(ctx).EventManager().EmitTypedEvent(&core.EventMintBatchCredits{
+	if err := sdk.UnwrapSDKContext(ctx).EventManager().EmitTypedEvent(&core.EventMint{
 		BatchDenom: batch.Denom,
+		Issuer:     issuer.String(),
```

Comment:
could also just do `req.Issuer` to avoid the `String` call

Reply:
https://github.com/regen-network/regen-ledger/pull/1086/commits/86c6b6de1c0fe529a3a2537f091ba878564692fc